bindings/examples/python/main.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+from __future__ import print_function
+from typing import Any, Dict, List, Optional, Set, Tuple
+import time
+import base64
+import io
+import yaml
+import sys
+import json
+import swagger_client
+from swagger_client import models
+from pathlib import Path
+from swagger_client.rest import ApiException
+from pprint import pprint
+from determined_common import context, yaml, constants
+
+
+SERVER_ADDRESS = "http://localhost:8080"
+SAMPLE_EXPERIMENT_DIR = Path.home().joinpath("projects/da/determined/e2e_tests/tests/fixtures/no_op")
```

Comment:
hardcoded path?

Reply:
this file is code samples I had for each of the languages we think we or external users would use. I'd probably leve the examples directory out of the pr

Reply:
yeah let's do that.  Until we fix this issue with the rest api where you have to send a jsonized tarball to create an experiment, I don't think we should be thinking of the rest api as consumable by external users.

And I desperately do not want to teach external users how to do that because that would make it harder for us to fix.